client/src/Data/Ellie/KeyCombo.elm

```diff
@@ -9,6 +9,28 @@ type alias KeyCombo =
     }
 
 
+controlKeysCodes : List Int
+controlKeysCodes =
+    [ 91, 92, 93, 224 ]
```

Comment:
Can we use `Set Int`s and `Set.member` for these? I don't the lookup to that much faster but I find `Set.member code` to be more clear about intention than `List.any ((==) code)`